### PR TITLE
deploy: update node-driver-registrar to latest version

### PR DESCRIPTION
**Pull request by humblec**
This commit update the node driver registrar to the latest
available version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

File: build.env [hunk 1]
```diff
@@ -52,7 +52,7 @@ CSI_ATTACHER_VERSION=v3.4.0
 CSI_SNAPSHOTTER_VERSION=v5.0.1
 CSI_PROVISIONER_VERSION=v3.1.0
 CSI_RESIZER_VERSION=v1.4.0
-CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.4.0
+CSI_NODE_DRIVER_REGISTRAR_VERSION=v2.5.1
```
Comment (Madhu-1) [inline build.env:55]: why update only the build.env, not the deployment yaml files?

**Review by Madhu-1 — CHANGES_REQUESTED** [review]

File: build.env (hunk 1 above)
Comment (humblec) [reply build.env:55]: Huh.. missed to commit.. :( .updating

**Review by Madhu-1 — DISMISSED** [review]

**Comment by nixpanic** [comment]
the `ci/retry/e2e` label only makes sense when all approvals are there, and there are no other PRs in the queue. Please do not add it when not required (yet).

File: deploy/rbd/kubernetes/csi-rbdplugin.yaml [hunk 2]
```diff
@@ -29,7 +29,7 @@ spec:
           securityContext:
             privileged: true
             allowPrivilegeEscalation: true
-          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.4.0
+          image: registry.k8s.io/sig-storage/csi-node-driver-registrar:v2.5.1
```
Comment (nixpanic) [inline deploy/rbd/kubernetes/csi-rbdplugin.yaml:32]: this commit misses the update for the NFS-provisioner

**Review by nixpanic — CHANGES_REQUESTED** [review]

File: deploy/rbd/kubernetes/csi-rbdplugin.yaml (hunk 2 above)
Comment (humblec) [reply deploy/rbd/kubernetes/csi-rbdplugin.yaml:32]: @nixpanic done.. thanks.. ptal.

**Comment by humblec** [comment]
@nixpanic comments are addressed .ptal. thanks

**Review by nixpanic — APPROVED** [review]

**Review by Madhu-1 — APPROVED** [review]

**Review by yati1998 — APPROVED** [review]

**Comment by humblec** [comment]
@mergifyio refresh

**Comment by nixpanic** [comment]
@Mergifyio requeue

**Comment by humblec** [comment]
Hmmm.. 2 days to get the PR in ,, still not !!

**Comment by humblec** [comment]
@Mergifyio refresh

**Comment by humblec** [comment]
@Mergifyio rebase

**Pull request merged**